apps/studio/components/interfaces/Database/Functions/CreateFunction/CreateFunctionHeader.tsx

```diff
@@ -0,0 +1,65 @@
+import { X } from 'lucide-react'
+
+import { SheetClose, SheetHeader, SheetTitle, cn } from 'ui'
+
+export const CreateFunctionHeader = ({
+  selectedFunction,
+  assistantVisible,
+  setAssistantVisible,
+}: {
+  selectedFunction?: string
+  assistantVisible: boolean
+  setAssistantVisible: (v: boolean) => void
+}) => {
+  return (
+    <SheetHeader
+      className={cn(
+        selectedFunction !== undefined ? 'pt-3 pb-0' : 'py-3',
+        'flex flex-row justify-between items-center border-b-0'
+      )}
+    >
+      <div className="flex flex-row gap-3 items-center max-w-[75%]">
+        <SheetClose
```

Comment:
Not using `showClose` prop you added?

Reply:
yeah i'd opt to keep the close button standard like this 
![image](https://github.com/supabase/supabase/assets/19742402/0e7bbe72-e94d-4202-a92b-036b55c365d8)

instead of this
![image](https://github.com/supabase/supabase/assets/19742402/aaf8cacc-6947-46e9-91b1-97bb2aca9171)

will make the changes but lemme know if anyone disagrees!

Reply:
yup, I tried to make it look like the RLS panel.